### PR TITLE
Adapted to multiclass classification (for language detection task)

### PR DESCRIPTION
**Pull request by kategimranova**
By default, binary classification will be performed (if argparse name "num_classes" is not defined)

### CODE REVIEW

**Review by SpirinEgor — CHANGES_REQUESTED** [review]
Good job! Let's prettify this a little bit:
- Define the type of tensor for labels right in the Dataset class (both TTR and VTR)
- Avoid passing `model_config` to train method

File: src/utils/train.py [hunk 1]
```diff
@@ -139,13 +141,14 @@ def evaluate_model(
...
-        predictions.append((output > 0).to(torch.float).cpu().detach())
+        predictions.append(torch.argmax(output, dim=1).cpu().detach())
         ground_truth.append(true_labels.cpu().detach())
 
     ground_truth = torch.cat(ground_truth).numpy()
     predictions = torch.cat(predictions).numpy()
-
-    precision, recall, f1_score, _ = precision_recall_fscore_support(ground_truth, predictions, average="binary")
+    
+    average = "binary" if num_classes == 2 else "micro"
```
[the remark this answers is not in the document]
Comment (kategimranova) [reply src/utils/train.py:149]: I decided to get num_classes as `num_classes = model.classifier.out_features` 'cause, the way I see it, it's more intuitive than `output.shape[-1]` (please correct me if I'm wrong)

File: src/models/vtr/classifier.py [hunk 2]
```diff
@@ -61,7 +62,7 @@ def __init__(
         )
         self.encoder = nn.TransformerEncoder(encoder_layer, num_layers=num_layers)
         self.norm = nn.LayerNorm(hidden_size)
-        self.classifier = nn.Linear(hidden_size, 1)
+        self.classifier = nn.Linear(hidden_size, num_classes)
```
[the remark this answers is not in the document]
Comment (kategimranova) [reply src/models/vtr/classifier.py:65]: As I understand, you mean this line: https://github.com/kategimranova/vitrina/blob/0e23354fa6258a3da70fde54137bb1331d1af130/src/models/vtr/classifier.py#L78
Now number of classes is always >= 2, so it works properly, because `squeeze(1)` is just doing nothing

File: src/utils/train.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SpirinEgor) [reply src/utils/train.py:149]: This is a very good solution!
I only may suggest here a more naive way. Save `num_classes` to a model property: `self.num_classes = num_classes` and then use it here as `model.num_classes`. This way, you reduce connectivity between classes and ensure invariant about a correct number of classes.

**Review by SpirinEgor — CHANGES_REQUESTED** [review]
The implementation is good, but don't forget to make proper changes and for TTR pipeline (loss and model)

**Review by SpirinEgor — APPROVED** [review]
Great job, thank you!
